functions.php

```diff
@@ -414,7 +414,9 @@ function render_navigation_block(array $attributes): string
     // For parent pages, get the previous and next siblings in the menu order
     $output = '';
     $siblings = array_filter($menu_items, function ($item) use ($nav_menu_item) {
-        return $item->menu_item_parent === $nav_menu_item->menu_item_parent;
+        return $item !== null
+            && $nav_menu_item !== null
+            && $item->menu_item_parent === $nav_menu_item->menu_item_parent;
```

Comment:
Can we make it simpler?
```suggestion
        return $item && $nav_menu_item && $item->menu_item_parent === $nav_menu_item->menu_item_parent;
```

Reply:
In the last functions.php code refactoring, the above code was moved to the `src/Post.php` file. @Osong-Michael  has opened a [new PR](https://github.com/greenpeace/planet4-master-theme/pull/2536/files) for this fix, so I’ve skipped this change in the current PR.